hs_core/hydroshare/resource.py

```diff
@@ -701,7 +701,10 @@ def delete_resource(pk):
     Note:  Only HydroShare administrators will be able to delete formally published resour
     """
 
-    res = utils.get_resource_by_shortkey(pk)
+    res = utils.get_resource_by_shortkey(pk, False)
```

Comment:
Not sure what's the reason for this change. I think the previous code was raising exception if it can't find the resource to be deleted - which I think reasonable.